app/lib/actions/katello/host/hypervisors.rb

```diff
@@ -0,0 +1,22 @@
+module Actions
+  module Katello
+    module Host
+      class Hypervisors < Actions::EntryAction
+        def plan(environment, content_view, hypervisor_params)
+          sequence do
+            hypervisor_results = plan_action(::Actions::Candlepin::Consumer::Hypervisors, hypervisor_params)
+            return if hypervisor_results.error
```

Comment:
you can't actually check for errors like this in the plan phase.  The run phase hasn't happend yet in ::Actions::Candlepin::Consumer::Hypervisors

If an error does occur, you may want to tell this entire action to skip the rest of the run phases and result in a stopped/errored task.  Heres an example: 
https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/sync.rb#L59-L61